### PR TITLE
Rebrand to MoveIt Pro

### PR DESCRIPTION
**Pull request by sea-bass**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get update && apt-get upgrade -y && \
     apt-get install -y --no-install-recommends ros-humble-rmw-cyclonedds-cpp
 
-# Set up the MoveIt Studio SDK workspace
+# Set up the MoveIt Pro SDK workspace
 ENV STUDIO_SDK_WS /opt/moveit_studio_sdk_ws
```
Comment (sjahr) [inline Dockerfile:11]: What about these ones? Will they be renamed later?

File: README.md [hunk 2]
```diff
@@ -65,4 +65,4 @@ The following command would move the robot arm to the `Look at Left Table` waypo
 ros2 run moveit_studio_py move_to_waypoint.py "Look at Left Table"
```
Comment (sjahr) [inline README.md:65]: Will the package names across the studio repos be renamed in a different PR

**Review by sjahr — APPROVED** [review]
I assume this is for now just a rebranding in the documentation?

File: Dockerfile (hunk 1 above)
Comment (sea-bass) [reply Dockerfile:11]: We won't rename env variables or package names in this this PR.

@eholum tried this in another PR and it was... 6k lines of changes.

Comment (eholum) [reply Dockerfile:11]: #NeverForget
